### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by wowtor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,16 +58,17 @@ Command-line interface
...
+2. run `lir <yaml file>`.
 
 The `examples` folder may be a good starting point for setting up an experiment.
 
 The elements of the experiment configuration YAML are looked up in the registry. The following lists all available
 elements in the registry.
 
 ```commandline
-pdm run benchmark --list-registry
+lir --list-registry
```
Comment (PimMeulensteen) [inline README.md:71]: Dit commando werkt niet, omdat het argument `FILENAME` verplicht is.

Comment (JoBraNFI) [reply README.md:71]: Goed gevonden! De huidige `lir` CLI commands blijken inderdaad helaas niet te werken op `main`. Hier een PR die dat oplost: https://github.com/NetherlandsForensicInstitute/lir/pull/50

Comment (wowtor) [reply README.md:71]: PR #50 gaat terug naar --setup, maar dat staat op zich los van het verplicht maken van de filename.

Comment (wowtor) [reply README.md:71]: Hij doet het nu. Zie unit tests.

File: lir/main.py [hunk 2]
```diff
@@ -61,11 +61,12 @@ def main() -> None:
         'setup',
         metavar='FILENAME',
         help='path to YAML file describing the evaluation setup',
+        nargs='?',
     )
     parser.add_argument(
         '--experiment',
         help='run a specific experiment (defaults to all experiments)',
-        nargs='*',
+        action='append',
```
Comment (wowtor) [inline lir/main.py:69]: dus niet meer `--experiment exp1 exp2`
maar: `--experiment exp1 --experiment exp2`

File: tests/test_main.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+@pytest.mark.parametrize("command", [
+    "--list-registry",
+    f"--list-experiments {config}",
+    f"{config}",
+    f"--experiment exp1 {config}",
+    f"--experiment exp1 --experimen exp2 {config}",
+])
+def test_main_commands(command: str):
+    main(command.split())
```
Comment (JoBraNFI) [inline tests/test_main.py]:
```suggestion
    try:
        main(command.split())
    except Exception:
        pytest.fail('Unexpected failure while parsing valid CLI arguments')
```

Misschien deze nog even expliciet laten falen wanneer er errors zijn met een uitleg

File: lir/data/data_strategies.py [hunk 4]
```diff
@@ -17,15 +17,14 @@ def __init__(self, source: DataSet, test_size: float | int, seed: int | None = N
...
 
     def __iter__(self) -> Iterator:
         """Allow iteration by looping over the resulting train/test split(s)."""
         instances = self.source.get_instances()
 
         indexes = np.arange(len(instances))
-        indexes_train, indexes_test, labels_train, labels_test = sklearn.model_selection.train_test_split(
-            indexes, instances.labels, test_size=self.test_size, shuffle=self.shuffle, random_state=self.seed
+        indexes_train, indexes_test = sklearn.model_selection.train_test_split(
+            indexes, stratify=instances.labels, test_size=self.test_size, shuffle=True, random_state=self.seed
```
Comment (JoBraNFI) [inline lir/data/data_strategies.py:27]: Deze change staat wat mij betreft los van het fixen van de README

Comment (wowtor) [reply lir/data/data_strategies.py:27]: Klopt, maar hij is nodig voor de unittest van main. Staat wel netjes in een aparte commit.

**Review by JoBraNFI — APPROVED** [review]
